README.md

```diff
@@ -1,14 +1,50 @@
-
-Refer to the following documentation for the Syncfusion .NET MAUI ListView Control: 
-https://help.syncfusion.com/maui/listview/getting-started
-
-Check out this GitHub example for the Syncfusion .NET MAUI ListView Control: 
-https://github.com/syncfusion/maui-demos/tree/master/MAUI/ListView
+The [.NET MAUI ListView control](https://www.syncfusion.com/maui-controls/maui-listview?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples) is used to present lists of data in a vertical or horizontal orientation with different layouts virtually.A quick-start project that helps you to integrate a ListView control in the .NET MAUI application by adding Syncfusion MAUI ListView package from NuGet. This project also explains on how to select items in the list, change its layout, and add a header and footer.
 
 ## Project prerequisites
 Make sure that you have the compatible versions of [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/ ) with the Dot NET MAUI workload and [.NET Core SDK 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later version in your machine before starting to work on this project.
 
 ## How to run this application
-To run this application, you need to first clone the `getting-started-with-the-dotnet-maui-listview` repository and then open it in Visual Studio 2019. Now, simply build and run your project to view the output.
\ No newline at end of file
+To run this application, you need to first clone the `getting-started-with-the-dotnet-maui-listview` repository and then open it in Visual Studio 2019. Now, simply build and run your project to view the output.
+
+## Features and Benefits
+
+###  Data binding
+The ListView has data binding support to work out-of-the box with popular data sources such as List, ObservableCollection, and many more.
+
+###  Different layouts
+Data can be displayed in different layouts such as linear and grid. Both layouts are supported in the horizontal list view as well.
+
+### Linear layout
+The linear layout arranges items linearly in a single column vertically or a single row horizontally.
+
+### Grid layout
+The grid layout arranges items in a predefined number of columns and the number of columns can be changed.
+
+### Grouping
+A group is a collection of items that belong to a specific category. When grouping is applied, the data is organized into different groups based on key values.
+
+## Related Links
+[Learn More about .NET MAUI ListView](https://www.syncfusion.com/maui-controls/maui-listview?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples) <br/>
+[Download Free Trial](https://www.syncfusion.com/downloads/maui/confirm?utm_source=github&utm_medium=listing&utm_campaign=maui-listview-github-samples) <br/>

```

Comment:
Use the below link for download free trial:
https://www.syncfusion.com/downloads/maui

Reply:
Added correct link